six/modules/c++/six.sicd/tests/test_read_sicd_from_vts.cpp

```diff
@@ -0,0 +1,321 @@
+/* =========================================================================
```

Comment:
If calling vts is a pain, I wonder if it would be better to just instantiate the CSM plugin directly yourself in a test program and then you could easily call any method you want instead of jumping through weird vts hoops.  You could use `sys::DLL` to open the plugin.